b/bolt/bolt_rhel_ubi8.4.sh

```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bolt
+# Version       : v1.3.1
+# Source repo   : https://github.com/boltdb/bolt
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation vikas.gupta8@ibm.com
```

Comment:
Is the Maintainer value supposed to be "BulkPackageSearch Automation vikas.gupta8@ibm.com"?

Reply:
Removed BulkPackageSearch Automation.

Reply:
Commonly used format is _NAME <EMAIL>_. Please refer other scripts.